packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -49,30 +53,32 @@ export default class Reachability extends StatelessWebexPlugin {
    * @public
    * @memberof Reachability
    */
-  gatherReachability = () => {
+  gatherReachability = async () => {
     this.setup();
-    // Reset reachability result to empty to ensure no stale data
+
+    // Remove stored reachability results to ensure no stale data
+    if (window.localStorage && window.localStorage.removeItem) {
+      window.localStorage.removeItem(REACHABILITY.localStorage);
+    }
+    else {
+      LoggerProxy.logger.error('Reachability:index#gatherReachability --> Error in accessing LocalStorage.');
+
+      return {};
```

Comment:
Should we throw the error here instead of returning an empty object?

Reply:
I believe this error is just for debugging purposes later. I think the flow is to just return empty latency results but not prevent or stop the flow from happening. Correct me if I'm wrong @arun3528.